src/containers/DateTime.js

```diff
@@ -37,6 +37,7 @@ export function DateTime({ value, format }) {
 	}
 
 	const m = isNaN(value) || value > 9999999999 ? moment(value) : moment(value * 1000);
+	m.locale(window.navigator.language.slice(0, 2));
```

Comment:
The locale should be taken from the current language settings.

Reply:
@ateska We had a discussion about this with @petrKavulok , so does it mean that it should follow the language of the application (taken from i18n) and not the browser's language?

Reply:
Yes, it sounds to me logical that then I switch to En, I see english names on moths etc.

Also, the language is set initially based on the browser lang + possibilities of the app (aka available localizations), which is also "correct".

Reply:
@ateska just to be clearify if I understood correctly - 
- initially the language should be set by the browser language
- then if user changes the apps language, the date should respect it and adjust accordingly

is that correct?..

Reply:
From the perspective of this component, you should always respect the language selection visible in the top bar - which is the language used for localization.

Reply:
https://github.com/TeskaLabs/asab-webui/blob/b0487a8c7b082c7b77b58dcb407f8b587f4f2490/src/modules/i18n/dropdown.js#L31

`i18n.language` ... specifically

BTW - that language should be in the reduct store so that you don't introduce dependency on i18n.

... and if (and only if) the language is not found in the redux store, then use the language from the browser.